c_tests/tests/missing_ctrl_pt.c

```diff
@@ -0,0 +1,18 @@
+// Compiler:
+//   status: error
+//   stderr:
+//     ...
+//     ld.lld: error: ykllvm couldn't find the call to `yk_control_point()`
+//     ...
+
+// Check that the system crashes if the control point is missing.
+
+#include <stdlib.h>
+#include <stdbool.h>
+
+int main(int argc, char **argv) {
+  while (true) {
```

Comment:
Do we even need the `while` loop to produce the "no control point" error?

Reply:
I think it would be clearer not to have the loop in there: it suggests we test for the existence of a loop with a control point when, I think, we test only for the existence of a control point.

[Maybe a good test for the next PR is "control point not in a loop"?!]